// console.log("hoo")

// try{
//     console.log("try block start");
//     console.log(x);
//     console.log("try block end");
// }
// catch(e){
//     console.log("i am inside catch");
//     console.log("your err",e);
// }
// finally{
//     console.log("i will run every time");
// }

// try{
//     console.log(x);
// }
// catch(err){
//     throw new Error("bhai");
// }

// let errCode = 100;
// if(errCode == 100){
//     throw new Error("√ènvalid json");
// }
